core/arch/arm/mm/tee_pager.c

```diff
@@ -198,6 +199,29 @@ static void pager_unlock(uint32_t exceptions)
 	cpu_spin_unlock_xrestore(&pager_spinlock, exceptions);
 }
 
+static void inval_tlb_for_pages(vaddr_t *va, size_t nb_elts)
```

Comment:
These two functions should preferably not be located in this file as it's complicated enough as it is.